recipe/meta.yaml

```diff
@@ -13,12 +13,17 @@ source:
     - get_config_h_filename.patch
 
 build:
-  number: 1
+  number: 2
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   skip: true  # [win]
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy >=1.17                          # [build_platform != target_platform]
```

Comment:
Why is this version constraint on `numpy` here? Also shouldn't `numpy` be in `host` and `run` as well if it is needed?

Reply:
Sorry, hadn't refreshed Github and did not see you message before merging. The numpy constraint was to prevent a fallback to 1.16 by default, which was forcing the Python version to 2 and is not supported. I only added numpy here based on the instructions [here](https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/) for adding a arm64 build for OSX. Numpy is not a requirement of the package itself.

Reply:
No worries. Posted after it was merged

Then would drop `numpy` if it is not used here

Reply:
The failed build is [here](https://github.com/conda-forge/sysv_ipc-feedstock/runs/4385851190), where Python 2 was requested by numpy 1.16. If there is a better way to fix this, I can create a new PR.

Reply:
Being dropped in PR ( https://github.com/conda-forge/sysv_ipc-feedstock/pull/13 )